subsys/net/lib/Kconfig

```diff
@@ -47,4 +47,6 @@ source "subsys/net/lib/trickle/Kconfig"
 
```

Comment:
The commit `net: lib: kconfig: add prometheus library kconfig source` could be squashed to the commit that introduce the library sources, no need for a separate commit for that.